server/src/routes/callAutomationEvent.ts

```diff
@@ -28,11 +29,14 @@ router.post('/', async function (req, res) {
        * is from the calling SDK and the callConnectionId is mapped to the correlationId from the transcription
        * service. We need to store this mapping so that we can fetch the transcription later.
        */
-      CALLCONNECTION_ID_TO_CORRELATION_ID[callConnectionId] = {
-        serverCallId: serverCallId,
-        correlationId: CALLCONNECTION_ID_TO_CORRELATION_ID[callConnectionId as string]?.correlationId
-      };
       getTranscriptionManager().setCallConnection(callConnectionId, serverCallId);
+      console.log(getTranscriptionManager().getCallConnection(serverCallId));
+    } else if (type === 'Microsoft.Communication.TranscriptionStarted') {
+      console.log('/automationEvent received', req.body);
```

Comment:
these needed, if they're helpful keep them in but be specfic on the event type as both say /automationEvent received

Reply:
These are left in to help the developer see what is happening since these events are streamed in from CallAutomation. I left the template of `automationEvent recieved` so that they will know their endpoint they sent them is working as expected. this is just left in since this is a sample and developers will try to run it locally.

Reply:
why not have `/automationEvent received - transcription started`, or move out of the if else and have `/automationEvent received, type`?

Reply:
So they can see the entire automation event, there is lots of good data in there if they want to do something else with the event.